src/stories/Quiz/Hooks/index.tsx

```diff
@@ -105,7 +103,7 @@ export default function HooksTemplate(args) {
     }
 
     const { currentQuestion } = state.quiz;
-    const currentQuestionData = state.quiz.currentQuestion.next_question;
+    const currentQuestionData = state?.quiz?.currentQuestion?.next_question;
     if (currentQuestion) {
```

Comment:
i'm not sure, but can we check for `currentQuestionData` here instead of `currentQuestion`, then we wouldn't need all of the `?.` in any of the nested JSX. we could even destructure  `currentQuestionData` after we have checked that it is defined in order to make the JSX more readable